ext/civicrm_admin_ui/managed/SavedSearch_Personal_Campaign_Pages.mgd.php

```diff
@@ -0,0 +1,267 @@
+                  'action' => '',
+                  'condition' => [],
+                ],
+                [
+                  'path' => 'civicrm/admin/pcp?action=revert&id=[id]',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Reject'),
+                  'style' => 'default',
+                  'condition' => [
+                    'status_id:name',
+                    'IN',
+                    [
+                      'Waiting Review',
+                      'Approved',
+                    ],
+                  ],
+                  'task' => '',
+                  'entity' => '',
+                  'action' => '',
+                  'join' => '',
+                  'target' => '_blank',
+                ],
+                [
+                  'path' => 'civicrm/admin/pcp?action=renew&id=[id]',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Approve'),
+                  'style' => 'default',
+                  'condition' => [
+                    'status_id:name',
+                    'IN',
+                    [
+                      'Waiting Review',
+                      'Not Approved',
+                    ],
+                  ],
+                  'task' => '',
+                  'entity' => '',
+                  'action' => '',
+                  'join' => '',
+                  'target' => '_blank',
+                ],
+                [
+                  'path' => '',
+                  'icon' => 'fa-toggle-on',
+                  'text' => E::ts('Enable'),
+                  'style' => 'default',
+                  'condition' => [],
+                  'task' => 'enable',
+                  'entity' => 'PCP',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                ],
+                [
+                  'task' => 'disable',
+                  'entity' => 'PCP',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-toggle-off',
+                  'text' => E::ts('Disable'),
+                  'style' => 'default',
+                  'path' => '',
+                  'action' => '',
+                  'condition' => [],
+                ],
+                [
+                  'task' => 'delete',
+                  'entity' => 'PCP',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-trash',
+                  'text' => E::ts('Delete'),
+                  'style' => 'danger',
+                  'path' => '',
+                  'action' => '',
+                  'condition' => [],
+                ],
+              ],
+              'type' => 'menu',
+              'alignment' => 'text-right',
+            ],
+          ],
+          'actions' => TRUE,
+          'classes' => [
+            'table',
+            'table-striped',
+          ],
```

Comment:
To be consistent with others pages, it is good to add a css rule to greyed disabled rows:
```
'cssRules' => [
            [
              'disabled',
              'is_active',
              '=',
              FALSE,
            ],
          ],
```